script/ci-release-build.js

```diff
@@ -0,0 +1,56 @@
+const args = require('minimist')(process.argv.slice(2))
+const assert = require('assert')
+const request = require('request')
+
+const ciJobs = [
+  'electron-linux-arm64',
+  'electron-linux-ia32',
+  'electron-linux-x64',
+  'electron-linux-arm'
+]
+
+const CIcall = (buildUrl, targetBranch, job) => {
+  console.log(`Triggering CircleCI to run build job: ${job} on branch: ${targetBranch} with release flag.`)
+
+  request({
+    method: 'POST',
+    url: buildUrl,
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    },
+    body: JSON.stringify({
+      'build_parameters': {
+        'RUN_RELEASE_BUILD': 'true',
+        'CIRCLE_JOB': job
+      }
+    })
+  }, (err, res, body) => {
+    if (!err && res.statusCode >= 200 && res.statusCode < 300) {
+      const build = JSON.parse(body)
+      console.log(`Check ${build.build_url} for status. (${job})`)
```

Comment:
This seems safe due to the statusCode check, but just to be sure -- are we certain that `build.build_url` will always be usable here?

Reply:
`build_url` should always be available here, if that's somehow not the case, it will show up as `undefined` but won't throw an error. The reason that happened probably won't be able to be conveyed here and should warrant further investigation anyway.